README.md

```diff
@@ -1,143 +1,170 @@
 # Web Content Accessibility Guidelines Summary
 
-This document will help you quickly get up to speed with Web Content Accessibility Guidelines 2.1 (WCAG), and avoid common accessibility-related mistakes.
+This document will help you quickly get up to speed with the [Web Content Accessibility Guidelines 2.1](https://www.w3.org/TR/WCAG21/) (WCAG), and avoid common accessibility-related mistakes.
 
 Click on each guideline to see details and code examples.
 
-This summary is a simplification. If you're unsure about how to meet a requirement, check the [official Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/).
+This summary is a simplification. If you're not sure how to meet a requirement, check the official [Web Content Accessibility Guidelines 2.1](https://www.w3.org/TR/WCAG21/).
+
+---
 
 ### 1. Easy to perceive
 
-Your website/app must present information in ways people can recognise and use, no matter how they consume content (for example if they have low vision or use a screen reader).
+Your website/app must present information in ways people can recognise and use, no matter how they consume content (for example if they have low vision, or use assistive technologies like a screen reader).
 
 #### Provide text alternatives for images
 
-* [Text alternatives for images](guidelines/1.1.1.md): Provide a text description for images. Make sure the description serves the same purpose as the image.
+* [Text alternatives for images](guidelines/1.1.1.md): Provide a text description for images. Make sure the description conveys the same message or functionality.
+
+<!-- Note to reviewer: I'm experimenting with adding this bullet point, in spite of the fact that we want to keep the document as short as possible. Because this is one of the most frequent accessibility mistakes on the web, and a pain to screen reader users as they use the image's file name every time this happens. Also, this requirement is part of the success criterion, but is not included in the bullet point above. -->
+* [Decorative images](guidelines/1.1.1.md): If an image doesn't convey any information, make sure that this is indicated in code.
```

Comment:
This bullet point breaks our new convention of making sure that the page that is linked to has the same heading and summary as the section it came from on the overview :(

Reply:
I was imagining that we would have 1 details page for 'Text alternatives for images', and another one for 'Decorative images'.

These two requirements are both summarised in 1.1.1 in the W3C guidelines, but they are different enough that we could make them 2 bullet points, with 2 detail pages.

This has downsides for sure (i.e. increased page length, and 2 detail pages linking to 1.1.1.). But this 'decorative images' requirement is largely unknown and causes lots of problems on the Web.

The alternative to having two bullet points would be to roll the point about decorative images into the bullet point above. We could. It'd just be quite long.

What do you think?